src/renderer/store/modules/profiles.js

```diff
@@ -89,6 +89,33 @@ const actions = {
     commit('setProfileList', profiles)
   },
 
+  async updateChannelThumbnail({ getters, dispatch }, { channelThumbnailUrl, channelName, channelId }) {
+    const profileList = getters.getProfileList
+    for (let i = 0; i < profileList.length; i++) {
```

Comment:
Is it possible to use `forEach` to replace for loop?
I only see `i` being used in `profileList[i]`

Reply:
it uses a break statement so i can't use a foreach

Reply:
you can use for-of in this case, if you want to:

```js
for (const profile of profileList) {
    // do stuff here with the profile variable
}
```